src/modules/mc_pos_control/CMakeLists.txt

```diff
@@ -40,6 +40,8 @@ px4_add_module(
 		PositionControl.cpp
 		Utility/ControlMath.cpp
 	DEPENDS
-		platforms__common
-	)
-# vim: set noet ft=cmake fenc=utf-8 ff=unix :
+		controllib
+		flight_tasks
+		git_ecl
```

Comment:
Are both git_ecl and ecl needed?

Reply:
At the moment yes. The git_ecl target is on the Firmware side and ensures the submodule is actually in place. Then ecl is the actual library in the ecl project. Mixing submodule dependencies into the build system like this is a bit unusual, but has worked surprisingly well. 

Later we can split the ecl project into separate libs for ekf, tecs, etc that build standalone or within PX4.

Reply:
Ok, it's just not obvious for someone adding a new module